usr/src/uts/common/vm/vm_page.c

```diff
@@ -5685,7 +5685,8 @@ page_migrate(
 	}
 }
 
-#define	MAX_CNT	60	/* max num of iterations */
+uint_t reclaim_maxcnt = 60; /* max total iterations */
+uint_t reclaim_nofree_maxcnt = 3; /* max iterations without progress */
```

Comment:
Nit - since these are global tunables, IMHO its better to add a prefix like - page_reclaim_max_retries and page_reclaim_nofree_retries ?